dashboard/app/assets/javascripts/utils.js.erb

```diff
@@ -10,9 +10,14 @@ var addClickTouchEvent = function(element, handler) {
 };
 
 function onURLBlocked(base_url, callback) {
-  $("<img width=0 height=0>")
+  $('<img width="0" height="0">')
```

Comment:
Maybe I'm not parsing exactly what's going on here, but if you've given an inline height/width of 0, do you need to set those properties with jQuery on lines 20 and 21?